tests/Opus/LanguageTest.php

```diff
@@ -86,6 +94,23 @@ public function testStoreLanguage()
         $this->assertEquals('0', $lang->getActive());
     }
 
+    public function testDeleteLanguage()
```

Comment:
Hier hat sich anscheinend das Verhalten verändert. Bei der Instanzierung mit einer ungültigen ID wurde in der Vergangenheit die Exception `ModelNotFoundException`  geworfen. Das ist hier nicht mehr der Fall. Ich fix das.